install.sh

```diff
@@ -1,4 +1,17 @@
 #!/usr/bin/env bash
+
+skip_user_prompt=0
+
+while [[ -n $1 ]]; do
+  case $1 in
+    -n | --no-user-prompt)
```

Comment:
Removing `-n` per my other comment.
```suggestion
    --no-user-prompt)
```

Also, this part is likely to clash with the upcoming PR: #615